benchmarking/commons/hpo_main_common.py

```diff
@@ -126,6 +132,7 @@ def parse_args(
     args = parser.parse_args()
     args.save_tuner = bool(args.save_tuner)
     args.scale_max_wallclock_time = bool(args.scale_max_wallclock_time)
+    args.use_long_tuner_name_prefix = bool(args.use_long_tuner_name_prefix)
```

Comment:
The key `use_long_tuner_name_prefix` is somewhat unclear in my opinion.

What if we instead create a flag like `custom-tuner-name-prefix`, which would be false by default. We would allow customers to pass their own prefixes and override our default value if they choose. 

As for what the default value should be, it could either: 1) be the "long" prefix that you propose in this CR (this would be a breaking change) or 2) be the status-quo prefix.

Reply:
But the prefix is a combination of other information, it is not a fixed string you can pass here.